vuepress/docs/next/tutorials/samples/install-standard-demo.md

```diff
@@ -82,23 +95,20 @@ The Entando Standard Banking Demo application demonstrates a number of the major
 
 * Keycloak integration for role based access controls
 * Micro frontends implemented using React and Angular and co-existing on the same dashboard page
```

Comment:
implementation? to match the other bullet point formats

Reply:
that would be a little awkward given the context and meaning

Reply:
Not a big deal but to clarify: Micro frontends implementation using React and Angular, co-existing on the same dashboard page